api/internal/tests/views/test_account_viewset.py

```diff
@@ -1109,7 +1214,7 @@ def test_update_sentry_plan_non_sentry_user(
         assert res.json() == {
             "plan": {
                 "value": [
-                    "Invalid value for plan: users-sentrym; must be one of ['users-free', 'users-basic', 'users-pr-inappm', 'users-pr-inappy']"
+                    "Invalid value for plan: users-sentrym; must be one of ['users-basic', 'users-pr-inappm', 'users-pr-inappy']"
```

Comment:
Ah this changed cause in the new available_plans we conditionally add users-free right?